### PR TITLE
T1 users/temp

### PR DESCRIPTION
**Pull request by Xzh0u**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/** @type {import('tailwindcss').Config} */
```
Comment (rayli09) [inline tailwind.config.js:1]: whats this file for?

File: src/Components/UserSidebar/Sidebar.js [hunk 2]
```diff
@@ -24,8 +24,8 @@ function Sidebar() {
           );
         })}
       </ul>
-      <hr />
+      {/* <hr /> */}
```
Comment (rayli09) [inline src/Components/UserSidebar/Sidebar.js:27]: rm

File: tailwind.config.js (hunk 1 above)
Comment (KenXiong123) [reply tailwind.config.js:1]: css framework
cc: @Xzh0u

**Review by KenXiong123 — APPROVED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (Xzh0u) [reply tailwind.config.js:1]: Yes. This is a config file of tailwindcss. tailwindcss is a popular CSS framework. If using this framework, then we don't need to write .css files manually. We only need to add some items in className. @rl3250

**Pull request merged**